erpnext/regional/india/e_invoice/einvoice.js

```diff
@@ -13,13 +13,15 @@ erpnext.setup_einvoice_actions = (doctype) => {
 
 			const { doctype, irn, irn_cancelled, ewaybill, eway_bill_cancelled, name, __unsaved } = frm.doc;
 
+			const automate_irns = await frappe.db.get_single_value('E Invoice Settings', 'automate_irns');
```

Comment:
if automate_irns is true user doesn't get to use generate e-way bill dialog to generate the e-way bill after submitting the doc which is possible behavior and is used in the case of the manual IRN generation process.